skt/reporter.py

```diff
@@ -26,6 +26,7 @@
 
 import skt
 import skt.runner
+from skt.misc import join_with_slash, get_patch_name, get_patch_mbox
```

Comment:
Is it possible to put this line above the runner import, to keep the alphabetical order?